examples/src/main/java/com/basistech/rosette/examples/RecordSimilarityExample.java

```diff
@@ -97,7 +97,7 @@ private void run() throws IOException {
                 ).build();
 
         HttpRosetteAPI rosetteAPI = new HttpRosetteAPI.Builder()
-                                    .key(getApiKeyFromSystemProperty())
+                                    .key("b1db524b37ad4b75f86729999d6017fd")
```

Comment:
Are You sure?

Reply:
@nnagyprecognoxcom I didn't mean for that to get left here, good catch